aws-xray/src/test/java/io/opentelemetry/contrib/awsxray/AwsSpanMetricsProcessorTest.java

```diff
@@ -56,6 +61,18 @@ private enum ExpectedStatusMetric {
 
   private AwsSpanMetricsProcessor awsSpanMetricsProcessor;
 
+  static class ThrowableWithMethodGetStatusCode extends Throwable {
```

Comment:
Lets add a test covering a Throwable with a `statusCode` method and a Throwable with neither `statusCode` nor `getStatusCode` methods.